.pipelines/templates/acr-cleanup.yaml

```diff
@@ -1,8 +1,10 @@
 steps:
   - bash: |
-      az login --identity
-      echo ${CHECKOUT_TAG}
-      az acr repository delete -n $(registry.name) --image oss/openservicemesh/osm-chart:${CHECKOUT_TAG} --yes
+      if [[ -n "${CHECKOUT_TAG}" ]]; then
```

Comment:
Can you log some sort of output to let the user know this step didn't happen and how they can fix it?

Reply:
Added error message if checkout_tag is not specified

Reply:
Added error message if checkout_tag variable fails to be created.

Reply:
I would change this to "Helm chart was not published to staging ACR because $CHECKOUT_TAG was not set by the pipeline"

However, this is not blocking so I'll approve.